core/src/main/java/org/jdbi/v3/core/statement/ParsedParameters.java

```diff
@@ -78,4 +79,28 @@ public String toString() {
             + ", parameterNames=" + parameterNames
             + '}';
     }
+
+    /**
+     * A static factory of named {@link ParsedParameters} instances.
+     * List given in argument must contain parameter names from SQL
+     * (i.e. variables preceded by a colon).
```

Comment:
I think this could be dubiously interpreted as having to include the colon in the names contained in the list

Reply:
I suggest:

> A static factory of named {link ParsedParameters} instances. List given in argument must contain parameter names from SQL. Names from a list must be stripped of a colon which stands before variable in unprocessed SQL.

Reply:
If I'm not mistaken, there's only talk of a colon at all because that's just the syntax jdbi uses by default with the ColonPrefixSqlParser, not because of a jdbc standard or anything. So the text should be agnostic about that.

> List must contain parameter names from the related statement. They must be "bare" names, independent of SQL variable processing syntax such as a prefixed colon or other delimiters.